src/amo/components/AddonBadges/index.js

```diff
@@ -64,6 +64,12 @@ export const AddonBadgesBase = (props: Props) => {
           label={i18n.gettext('Not compatible with Firefox Quantum')}
         />
       ) : null}
+      {addon.requires_payment ? (
+        <Badge
+          type="requires-payment"
+          label={i18n.gettext('Some features may require payment')}
```

Comment:
If the add-on has "requires payment" flagged, we shouldn't be so wishy-washy about "may require" 😄

I think this should just say "Some features require payment". That's more accurate and less confusing. Any issues with that, @jvillalobos?

Reply:
This was discussed quite a bit in the issue, and I thought the final word was at https://github.com/mozilla/addons-frontend/issues/2080#issuecomment-366061040, but I guess it doesn't hurt to double check before landing. 😃

Reply:
The flag can mean a number of different things (see the expanded label on dev tools), so I think it's okay to keep the slightly vague wording.

Reply:
Ten four 👍